pkgs/applications/misc/worker/default.nix

```diff
@@ -1,15 +1,15 @@
-{pkgs, stdenv, xorg, fetchurl }:
```

Comment:
Please pass `libX11` instead of the entire `xorg` package set; `callPackage` should be able to fill it in.